site/content/docs/5.0/components/offcanvas.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: docs
+title: Offcanvas
+description: Build hidden sidebars into your project for navigation, shopping carts, and more with a few classes and our JavaScript plugin.
+group: components
+toc: true
+---
+
+## How it works
+
+The offcanvas JavaScript plugin shows and hides sidebar on the left, right, or bottom of your viewport. Buttons or anchors are used as triggers that are attached to specific elements you toggle.
```

Comment:
"shows and hides sidebar on the left..." > "shows and hides an off-canvas panel on the left..." (or something similar)